File: decompiled/sources/com/google/android/gms/fido/fido2/Fido2PendingIntent.java
package com.google.android.gms.fido.fido2;

import android.app.Activity;
import android.content.IntentSender;

public interface Fido2PendingIntent {
    boolean hasPendingIntent();

    void launchPendingIntent(Activity activity, int i) throws IntentSender.SendIntentException;
}

File: decompiled/sources/com/google/android/gms/fido/fido2/Fido2PendingIntent.kt
package com.google.android.gms.fido.fido2

import android.app.Activity
import android.content.IntentSender

interface Fido2PendingIntent {
    fun hasPendingIntent(): Boolean

    @Throws(IntentSender.SendIntentException::class)
    fun launchPendingIntent(activity: Activity, requestCode: Int)
}
